README.md

```diff
@@ -17,19 +17,15 @@ The exporter can be configured via configuration file and commandline parameters
 Usage of ./osquery_exporter:
   -config.file string
     	Config file (default "config.yaml")
-  -log.format value
-    	Set the log target and format. Example: "logger:syslog?appname=bob&local=7" or "logger:stdout?json=true" (default "logger:stderr")
-  -log.level value
-    	Only log messages with the given severity or above. Valid levels: [debug, info, warn, error, fatal] (default "info")
```

Comment:
As far as I can tell, these two flags don't seem to work - unless I'm missing something?